www/hockeyapp.js

```diff
@@ -20,6 +20,9 @@ var hockeyapp = {
     feedback: function (success, failure) {
         exec(success, failure, "HockeyApp", "feedback", []);
     },
+    feedbackModal: function (success, failure, takeScreenshot, data) {
```

Comment:
Sounds good. I've updated the PR with the changes to both the function and parameter names